packages/core/components/Modal/DataSourcePrompt/index.tsx

```diff
@@ -68,21 +68,7 @@ export default function DataSourcePrompt({ onDismiss }: Props) {
     const onEnterURL = throttle(
         (evt: React.FormEvent) => {
             evt.preventDefault();
-            const uriResource = dataSourceURL
```

Comment:
Separated the logic in lines 71-85 into a separate method since gets reused in DatasetRow and DatasetDetails